xarray/plot/plot.py

```diff
@@ -433,11 +470,16 @@ def newplotfunc(darray, x=None, y=None, ax=None, row=None, col=None,
         # This allows the user to pass in a custom norm coming via kwargs
         kwargs.setdefault('norm', cmap_params['norm'])
 
-        ax, primitive = plotfunc(xval, yval, zval, ax=ax,
-                                 cmap=cmap_params['cmap'],
-                                 vmin=cmap_params['vmin'],
-                                 vmax=cmap_params['vmax'],
-                                 **kwargs)
+        if 'imshow' == plotfunc.__name__ and isinstance(aspect, basestring):
+            # forbid usage of mpl strings
+            raise ValueError("plt.imshow's `aspect` kwarg is not available " \
```

Comment:
Just use implicitly string concatenation here instead of writing `\` (you almost never want/need to actually use `\`)